gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/Form.java

```diff
@@ -36,6 +37,15 @@
  */
 public class Form extends AbstractForm implements HasType<FormType> {
 
+    public Form() {
```

Comment:
It's the equivalent of calling Form(FormType.DEFAULT).

I like it better like this

~~~
public Form() {
    this(FormType.DEFAULT);
}

@UiConstructor
public Form(final FormType type) {
    setType(type);
}
~~~

Makes it a little more obvious.